File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Querying/IdListQueryResultTestCase.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.Internal.Query.Result;
using Db4objects.Db4o.Tests.Common.Querying;

namespace Db4objects.Db4o.Tests.Common.Querying
{
	public class IdListQueryResultTestCase : QueryResultTestCase
	{
		public static void Main(string[] args)
		{
			new IdListQueryResultTestCase().RunSolo();
		}

		protected override AbstractQueryResult NewQueryResult()
		{
			return new IdListQueryResult(Trans());
		}
	}
}

File: db4oj.tests/src/com/db4o/db4ounit/common/querying/IdListQueryResultTestCase.java
/* Copyright (C) 2006  Versant Inc.  http://www.db4o.com */

package com.db4o.db4ounit.common.querying;

import com.db4o.internal.query.result.*;


public class IdListQueryResultTestCase extends QueryResultTestCase {

	public static void main(String[] args) {
		new IdListQueryResultTestCase().runSolo();
	}

	protected AbstractQueryResult newQueryResult() {
		return new IdListQueryResult(trans());
	}

}
